src/main/kotlin/Menu.kt

```diff
@@ -0,0 +1,96 @@
+import java.util.*
+
+interface Menu {
+    val itemMake: String
+    val itemExit: String
+    val intro: String
+    val introInput: String
+    val inputName: String
+    val listItemsMenu: MutableList<String>
+
+
+    fun <K : String, V> prepareListItemMenu(map: MutableMap<K, V?>): MutableList<String> {
```

Comment:
🍏 String является финальным классом, от этого класса нельзя наследоваться. Значит, в данном случае в качестве K может выступать только String и ничего больше. Поэтому можно K вообще убрать. И тогда в строке 52 уже не нужен будет каст к String.
```suggestion
    fun <V> prepareListItemMenu(map: MutableMap<String, V?>): MutableList<String> {
```